packages/terra-i18n-plugin/package.json

```diff
@@ -21,15 +21,15 @@
   },
   "homepage": "https://github.com/cerner/terra-core#readme",
   "devDependencies": {
-    "babel-jest": "^19.0.0"
+    "babel-jest": "^19.0.0",
+    "enhanced-resolve": "^3.4.1",
+    "memory-fs": "^0.4.1",
+    "webpack": "^3.6.0"
```

Comment:
I'm concerned about pulling in / using webpack 3 in the i18n-plugin. I think we'll want to pull in webpack v2 in the i18n-plugin or update toolkit / terra-site to use webpack 3.

Reply:
you should be able to drop this back to the supported webpack version in terra-core

Reply:
updated it to match the webpack version listed under terra-site

Reply:
Why is webpack a devDependency at all?

Reply:
I use its compiler in the tests
const Compiler = require('webpack/lib/Compiler');